math/mathmore/src/Polynomial.cxx

```diff
@@ -157,6 +157,7 @@ const std::vector< std::complex <double> > &  Polynomial::FindRoots(){
       n--;
     }
 
+    std::vector< std::complex < double > > fRoots;
```

Comment:
I understand the intention, but maybe we should not use the `f` for "field" in this context, since it's not any more a field of the class.